src/h5web/visualizations/index.tsx

```diff
@@ -132,22 +132,8 @@ export const VIS_DEFS: Record<Vis, VisDef> = {
       const dataset = getLinkedEntity(signal, group, metadata);
       assertDefined(dataset, `Expected "${signal}" signal entity to exist`);
       assertDataset(dataset, `Expected "${signal}" signal to be a dataset`);
-
-      assertNumericType(
-        dataset.type,
-        `Expected "${signal}" dataset to have numeric type`
-      );
-      assertSimpleShape(
-        dataset.shape,
-        `Expected "${signal}" dataset to have simple shape`
-      );
-
-      const dimsCount = dataset.shape.dims.length;
-      if (dimsCount === 0) {
-        throw new Error(
-          `Expected "${signal}" dataset to have at least one dimension`
-        );
-      }
+      assertNumericType(dataset);
+      assertSimpleShape(dataset);
```

Comment:
Note that I'm no longer logging the signal's name. I feel like the debugging benefit is not sufficient to justify the clutter in the code. In the future, the assertion functions could take care of this, though (through a hypotetic `dataset.name` property -- on this note, I wonder whether we could set this property in `getLinkedEntity` and return an `HDF5NamedEntity` of sorts...)

Reply:
You could the same trick you used on other assertion functions: using the error message as parameter to log the signal name.

Anyhow, I think it is fine like this